web/src/lib/components/shared-components/profile-image-cropper.svelte

```diff
@@ -28,13 +28,13 @@
     canvas.height = img.height;
     const context = canvas.getContext('2d');
     if (!context) {
-      throw new Error('Could not get canvas context.');
+      throw new Error($t('errors.could_not_get_canvas_context'));
     }
     context.drawImage(img, 0, 0);
     const imageData = context.getImageData(0, 0, canvas.width, canvas.height);
     const data = imageData?.data;
     if (!data) {
-      throw new Error('Could not get image data.');
+      throw new Error($t('errors.could_not_get_image_data'));
```

Comment:
Messages that are only logged to console shouldn't be translated

Reply:
You're right, fixed. Thanks!